src/compat/dbtype_def.h

```diff
@@ -1265,6 +1265,13 @@ extern "C"
       unsigned int zone_id;	/* geographical zone id */
     };
   };
+
+  typedef enum
+  {
+    LOB_CREATE_DIR = 0,
+    LOB_DROP_TABLE_DIR = 1,
+    LOB_DROP_COLUMN_DIR = 2
+  } LOB_DIR_MANAGE_MODE;
```

Comment:
전체적인 코드 일관성을 위해 아마도 리뷰하는 동안 동일한 부분에 대해 여러번 수정 요청이 발생할 수 있을 것 같습니다. 이점 참고하시고 리뷰 내용 확인해 주세요.
```suggestion
  typedef enum
  {
    LOBDIR_CREATE = 0,
    LOBDIR_REMOVE = 1
  } LOBDIR_MODE;
```
추가적으로, LOB_DROP_COLUMN_DIR의 경우 전달 받은 attr_length로 구분 가능할 것 같아 제외시켰습니다.

Reply:
위에서 언급해주신 두 경우를 함수로 나누게 되어 LOB_DIR_MANAGE_MODE는 더 이상 사용하지 않도록 하였습니다.